packages/slice-machine/src/components/TextLink/TextLink.css.ts

```diff
@@ -0,0 +1,108 @@
+import { style, styleVariants } from "@vanilla-extract/css";
+
+import { colors, selectors, sprinkles } from "@prismicio/editor-ui";
+
+export const root = style([
+  sprinkles({
+    all: "unset",
+    cursor: "pointer",
+    display: "flex",
+    alignItems: "center",
+    color: {
+      ...colors.indigo11,
```

Comment:
Should be purple11 here, no?

Reply:
Oh it should not be specified here actually. Good catch!